tests/test_herzog.py

```diff
@@ -26,7 +37,62 @@ def test_parser(self):
 
         self.assertEqual(len(expected_cells), len(cells))
         for expected_cell, cell in zip(expected_cells, cells):
-            self.assertEqual(expected_cell, cell)
+
+            # convert all "source" to "string"; equally valid as a list or a string, example:
+            #     '# This is a header\ndoom and gloom\n\n## frank is a gangster\nevidence'
+            #     ['# This is a header\n', 'doom and gloom\n', '\n', '## frank is a gangster\n', 'evidence']
+            if isinstance(expected_cell.get('source', None), list):
+                expected_cell['source'] = ''.join(expected_cell['source'])
+
+            self.assertEqual(expected_cell, cell, f'\n{expected_cell}\n{cell}')
```

Comment:
I’m not sure I understand why this test was modified.

Reply:
I'm not sure if it's the ipynb version, but I believe I explained it in the comments.

Reply:
Is the comment meant to indicate that .ipynb formatted cells can contain either lists or strings in the `source` field? Can you provide a link to documentation describing this feature of the .ipynb format?

Currently, Herzog does not produce cells containing list formatted `source` fields. Until it does so, it seems more appropriate for the test to expect only string formatted `source` fields.

Unless I’m missing something?

Reply:
Please break this out in a separate PR.